documentation/modules/proc-configuring-kafka-bridge-tracing.adoc

```diff
@@ -95,6 +95,21 @@ OTEL_EXPORTER_ZIPKIN_ENDPOINT=http://localhost:9411/api/v2/spans <2>
 <1> The name of the tracing system. In this example, Zipkin is specified.
 <2> The endpoint of the specific selected exporter that listens for spans. In this example, a Zipkin endpoint is specified.
 
+== Supported properties
```

Comment:
I haven't had a section like this in the past. I am not sure we want it now otherwise we should list all span attributes (i.e. otel.status_code, span.kind, ....). I would take them out but of course taking what you have in the CHANGELOG mentioning new attributes and the ones deprecated and planned to be removed.

Reply:
I added them as it felt a bit random to just document them in the changelog with out mentioning them anywhere else. As much for discoverability as anything else. 

I only documented the bridge specific tags, which seems like a reasonable choice. We could add a sentence along the lines of `In addition to the open telemetry standard tags we include the following`.